packages/interactivity-router/src/index.ts

```diff
@@ -54,7 +54,10 @@ const navigationMode: 'regionBased' | 'fullPage' =
 
 // The cache of visited and prefetched pages, stylesheets and scripts.
 const pages = new Map< string, Promise< Page | false > >();
-const headElements = new Map< string, { tag: HTMLElement; text: string } >();
+export const headElements = new Map<
+	string,
+	{ tag: HTMLElement; text?: string }
+>();
```

Comment:
I've exported the `headElements` so that we don't have to pass it as an argument to `fetchHeadAssets()`.